.eslintrc.json

```diff
@@ -1,21 +1,37 @@
 {
     "env": {
-      "browser": true,
-      "es6": true,
-      "jest": true
```

Comment:
Kindly explain the reason why you change the content of this file, students are not allow to change the content.
![image](https://user-images.githubusercontent.com/31897434/72196970-87c84880-341c-11ea-9ad8-6942aec47717.png)

https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

Reply:
To learn eslintrc lint, I configured of own and I took microverse eslintrc as my base. Another important thing is, I didn't edit it for an escape from linters. Even If read that properly you will, it is same as microverse linter config written in a different way.

Up to this point, I didn't use jest, so I removed it. later in the project added.